examples/circle_packing.ml

```diff
@@ -96,12 +95,16 @@ let make_concentric circle =
 
 (* main fn *)
 let () =
-  init ~size:resolution ();
-  background (1., 1., 1., 1.);
-  set_line_width 0.001;
+  init ~size:(tmap int_of_float resolution) ();
+  background (255, 255, 255, 255);
+  set_line_width 1;
   let circles = pack_circles () in
   let circles = List.flatten (List.map make_concentric circles) in
   List.iter
-    (fun ((x, y), radius) -> draw_with_color (circle ~c:(point x y) radius))
+    (fun ((x, y), radius) ->
+      draw_with_color
+        (circle
+           ~c:(point (int_of_float x) (int_of_float y))
+           (int_of_float radius)))
```

Comment:
I can't use `Transform.scale` because I can't test the radii of the circles as they aren't exposed in the API. Also, that would require switching from the current representation of circles as (float * float) * float tuples.

I can make `make_concentric` and `pack_circles` take and return ints but that would require a lot of conversion back into floats because the math required to check if circles overlap is done with floats. 

I think that, in general, this sketch is illustrative of what an end-user writing a more complex sketch will look like.